src/Summary.jsx

```diff
@@ -56,87 +56,139 @@ class Summary extends React.Component {
-                </div>
-             </section>
+        <section className="sectionintern">
+          <div className="internheader" />
+          <div className="map">
+            <MapViewer />
+          </div>
+          <div className="blockdata">
+            <Link to="/indicatorsDash">
+              <button
+                type="button"
+                key="indBtn"
+                className="generalbtn absright"
+              >
+                indicadores
+              </button>
+            </Link>
+            <h1>Sobre el bloque</h1>
+            <div className="line" />
+            <h5 className="hectareas">
+              <b>{numberWithCommas(areaData.area)}</b>
+              {' '}
+              ha
+            </h5>
+            <div className="iconos">
+              <img
+                className={areaData.categories.find((item) => item === 1) ? '' : 'nogo'}
+                src={protegidas}
+                alt="Áreas protegidas"
+                title="Áreas protegidas"
+              />
+              <img
+                className={areaData.categories.find((item) => item === 2) ? '' : 'nogo'}
+                src={reservas}
+                alt="Reservas forestales"
+                title="Reservas forestales"
+              />
+              <img
+                className={areaData.categories.find((item) => item === 3) ? '' : 'nogo'}
+                src={estrategicos}
+                alt="Ecosistemas estratégicos"
+                title="Ecosistemas estratégicos"
+              />
+              <img
+                className={areaData.categories.find((item) => item === 4) ? '' : 'nogo'}
+                src={etnicas}
+                alt="Territorios étnicos"
+                title="Territorios étnicos"
+              />
+              <img
+                className={areaData.categories.find((item) => item === 5) ? '' : 'nogo'}
+                src={campesinas}
+                alt="Zonas de reserva campesina"
+                title="Zonas de reserva campesina"
+              />
+              <img
+                className={areaData.categories.find((item) => item === 6) ? '' : 'nogo'}
+                src={infraestructura}
+                alt="Proyectos e infraestructura"
+                title="Proyectos e infraestructura"
+              />
+              <img
+                className={areaData.categories.find((item) => item === 7) ? '' : 'nogo'}
+                src={ordenamiento}
+                alt="Ordenamiento"
+                title="Ordenamiento"
+              />
+            </div>
+            <p>{areaData.description}</p>
+            <h1>Biomas</h1>
+            <div className="line" />
+            <br />
+            <div>
+              {graphData1
+                ? RenderGraph(
+                  graphData1, '', '', 'SmallBarStackGraph',
+                  'Zoonobioma', ['#5f8f2c', '#fff'], null, null,
+                  '', '%',
+                ) : 'Cargando...'}
```

Comment:
According to [Aribnb styleguide](https://github.com/airbnb/javascript#comparison--nested-ternaries) ternaries shouldn't be multilne, it's better to use if/else statements.

But if you insist in use it, the colon should be at the [beginning of the line](https://github.com/airbnb/javascript/issues/1281#issuecomment-276191835)

Reply:
Fixed on 0f37063